client/src/components/Form/BlogsInputs/BlogsPreviewButton.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import PrimaryButton from '../../common/PrimaryButton';
+
+const BlogsPreviewButton = ({redirectToView}) => (
+    <PrimaryButton onClick={() => {redirectToView()}}>
+        Preview
+    </PrimaryButton>
+)
+
```

Comment:
this is unnecessary abstraction. You have already abstracted Primary Button now for each usage of primary button we start to build component imagine how many components we will have.

Reply:
It is looking unnecessary right now but let's say in future we want to replace the dependency of semantic, in that case, it would be useful